rustfmt.toml

```diff
@@ -0,0 +1,55 @@
+indent_style = "Block"
+use_small_heuristics = true
+binop_separator = "Front"
+combine_control_expr = true
+comment_width = 80
+condense_wildcard_suffixes = false
+control_brace_style = "AlwaysSameLine"
+disable_all_formatting = false
```

Comment:
are these all non-default? I think it would be sane to only specify non-default switches here, to avoid the temptation to deviate from those defaults

Reply:
These are all the defaults, I have just included them here so that you can see all the available options. I'll remove them when everything has been decided on.